mappings/net/minecraft/inventory/BasicInventory.mapping

```diff
@@ -18,11 +21,15 @@ CLASS net/minecraft/class_1277 net/minecraft/inventory/BasicInventory
 		ARG 1 stack
 	METHOD method_24514 clearToList ()Ljava/util/List;
 		COMMENT Clears this inventory and return all the non-empty stacks in a list.
+	METHOD method_24825 canCombine (Lnet/minecraft/class_1799;Lnet/minecraft/class_1799;)Z
+		ARG 1 one
+		ARG 2 two
 	METHOD method_5488 removeListener (Lnet/minecraft/class_1265;)V
 		ARG 1 listener
 	METHOD method_5489 addListener (Lnet/minecraft/class_1265;)V
 		ARG 1 listener
-	METHOD method_5491 add (Lnet/minecraft/class_1799;)Lnet/minecraft/class_1799;
+	METHOD method_5491 addStack (Lnet/minecraft/class_1799;)Lnet/minecraft/class_1799;
```

Comment:
What was wrong with `add`?

Reply:
maybe he has concern with method clash from collection

Reply:
Naming consistency with `Inventory`, actually. I've renamed it to be an overload of the other add methods.